### ISSUE TITLE
Scraping broken

### ISSUE THREAD

**Issue by seanocali** [lewishenson/FluentMetacritic#8]
I tried each of your sample code lines and I just get enumeration yield no results.  The site isn't down or anything.

**Comment by lewishenson** [comment]
Yes, it looks like the Metacritic's site has changed therefore breaking the scraping. Will try and look into this weekend (or if you've got a fix already, happy to accept a PR).

**Comment by lewishenson** [comment]
Not going to be as straightforward as I hoped, the structure has changed a fair bit. It even looks like "Features" have disappeared completely.

**Comment by seanocali** [comment]
Sorry about that.  I guess that's the way these things go.  It's a shame they don't have a public API.

**Comment by lewishenson** [comment]
It is a shame. I've just discovered it's against Metacritic's Terms of Use to scrape their site so I've decided to sunset FluentMetacritic and will not be fixing any issues. Sorry.
